lightningd/peer_control.c

```diff
@@ -550,10 +550,11 @@ static struct amount_sat commit_txfee(const struct channel *channel,
 			num_untrimmed_htlcs++;
 	}
 
-	/* Funder is conservative: makes sure it allows an extra HTLC
-	 * even if feerate increases 50% */
-	return commit_tx_base_fee(local_feerate + local_feerate / 2,
-				  num_untrimmed_htlcs + 1);
+	/*
+	 * Having 2-times feespike margin will become BOLT #2 standard by:
+	 * https://github.com/lightningnetwork/lightning-rfc/issues/740
+	 */
```

Comment:
Can we keep the original comment as well? It's better to have comments that refer to facts, not ones that refer to changes of those facts.

Reply:
Okay, but now the bolt checker will likely fail, as this specific PR 740 is not merged yet...

Reply:
afaik the bolt checker only checks comments that start with BOLT#XX, not comments that have a reference to the repo :-)

Reply:
thanks